src/ifcfg/__init__.py

```diff
@@ -24,6 +24,8 @@ def get_parser_class():
         Parser = parser.LinuxParser
         if not os.path.exists(Parser.get_command()[0]):
             Parser = parser.UnixIPParser
+            if not os.path.exists(Parser.get_command()[0]):
+                Log.warning("Neither `ifconfig` nor `ip` commands are available, listing network interfaces is likely to fail")
```

Comment:
This is an exact repetition of line 25?

Reply:
A direct copy-paste... 
But `Parser` is a different thing by then but I'll try to simplify the logic.

Reply:
With only options, I don't see a way to keep the logic as clear as it's now. When there's more options, iterating over them makes sense.
These utils works fine so adding more options is not a goal at all and thus this repeated condition is the cleanest way to achieve this IMHO.

Reply:
How did this check work anyways? `Parser.get_command()[0]` returned a `/` in cases where the command exists?

Reply:
My main questions is why the conditional `if` statement is an exact copy paste from the statement it's nested within.

> Parser.get_command()[0] returned a / in cases where the command exists?

It seems to extract the first character of the path, which is probably wrong or weird. You're welcome to make corrections. The tests should cover this :)

Reply:
> My main questions is why the conditional if statement is an exact copy paste from the statement it's nested within.

The code is the same but since `Parser` is assigned a different value in line 26, the effect is different. 
It's the same, because the same check should happen on the new Parser-value: does this new parser have a command that actually works?

> It seems to extract the first character of the path, which is probably wrong or weird. You're welcome to make corrections. The tests should cover this :)

That is fixed now.